### PR TITLE
removing event listener in index.html and doing this with css instead…

### PR DESCRIPTION
**Pull request by UncleFifi**
… and adding minor styles for gameplay making it look more like a SPA
Game's info pane now contains a border which will bring the attention of user stats for game instance.

### CODE REVIEW

File: js/info.js [hunk 1]
```diff
@@ -41,19 +41,23 @@ define([], function() {
 
     const regenerate = function() {
         ++totalSeconds;
-        document.getElementsByClassName("info")[0].innerText = getInfo();
+        // document.getElementsByClassName("info")[0].innerText = getInfo();
```
Comment (titivermeesch) [inline js/info.js:44]: Please don't keep these comments if they have no use anymore

**Review by titivermeesch — CHANGES_REQUESTED** [review]
Please check the merge conflict, it's a really small issue with the recent ES6 conversion.

File: js/info.js (hunk 1 above)
Comment (UncleFifi) [reply js/info.js:44]: ok, will not keep comments

**Comment by UncleFifi** [comment]
please keep as is with with arrow function

**Pull request closed without merging**